llvm/test/Support/interrupts.test

```diff
@@ -34,8 +35,8 @@ def run_wrapper():
                                 startupinfo=startupinfo,
                                 creationflags=subprocess.CREATE_NEW_CONSOLE)
     else:
-        proc = subprocess.Popen(args,
-                                stderr=subprocess.PIPE)
+        proc = subprocess.Popen(args, stderr=sys.stderr)
+        proc.wait()
```

Comment:
Are you sure `wait` is the right thing you want? Could we simplify both this and the line a couple of lines above to `subprocess.run`?

Reply:
> Are you sure `wait` is the right thing you want? Could we simplify both this and the line a couple of lines above to `subprocess.run`?

Yes, we can use `subprocess.run`, the PR changed accordingly.